overlord/snapstate/snapstatetest/devicectx.go

```diff
@@ -74,6 +75,14 @@ func (dc *TrivialDeviceContext) HasModeenv() bool {
 	return dc.Model().Grade() != asserts.ModelGradeUnset
 }
 
+func (d *TrivialDeviceContext) IsCoreBoot() bool {
+	return d.HasModeenv() || (!d.HasModeenv() && !release.OnClassic)
```

Comment:
here we should also not use release.OnClassic for consistency

Reply:
Right, changed now